internal/services/mssqlmanagedinstance/mssql_managed_instance_active_directory_administrator_resource_test.go

```diff
@@ -25,9 +26,6 @@ func TestAccMsSqlManagedInstanceActiveDirectoryAdministrator_basic(t *testing.T)
 	r := MsSqlManagedInstanceActiveDirectoryAdministratorResource{}
 
 	data.ResourceTest(t, r, []acceptance.TestStep{
-		{
-			Config: r.template(data),
-		},
```

Comment:
just want to check what the reason is for removing/changing this?

Reply:
Hi @catriona-m , thanks for your time. This is because `TestAccMsSqlManagedInstanceActiveDirectoryAdministrator_basic` fails with the following error.

![image](https://github.com/user-attachments/assets/6483826b-9c4a-4815-94f6-933fb2923799)

Reply:
thanks @sinbai. I notice we are getting a similar failure on another test, would it be within the scope of this PR to fix it too?

```
------- Stdout: -------
=== RUN   TestAccMsSqlManagedInstanceFailoverGroup_update
=== PAUSE TestAccMsSqlManagedInstanceFailoverGroup_update
=== CONT  TestAccMsSqlManagedInstanceFailoverGroup_update
    testcase.go:173: Step 1/9 error: Pre-apply plan check(s) failed:
        azurerm_mssql_managed_instance_failover_group.test - Resource not found in plan ResourceChanges
--- FAIL: TestAccMsSqlManagedInstanceFailoverGroup_update (21.08s)
FAIL
```

Reply:
> thanks @sinbai. I notice we are getting a similar failure on another test, would it be within the scope of this PR to fix it too?
> 
> ```
> ------- Stdout: -------
> === RUN   TestAccMsSqlManagedInstanceFailoverGroup_update
> === PAUSE TestAccMsSqlManagedInstanceFailoverGroup_update
> === CONT  TestAccMsSqlManagedInstanceFailoverGroup_update
>     testcase.go:173: Step 1/9 error: Pre-apply plan check(s) failed:
>         azurerm_mssql_managed_instance_failover_group.test - Resource not found in plan ResourceChanges
> --- FAIL: TestAccMsSqlManagedInstanceFailoverGroup_update (21.08s)
> FAIL
> ```

Hi @catriona-m I have fixed the test. It passed on locally by setting `env.ARM_TEST_LOCATION=	Westeurope` and `env.ARM_TEST_LOCATION_ALT = NorthEurope`. This means that `ARM_TEST_LOCATION` and `ARM_TEST_LOCATION_ALT` need to be paired regions. See the following `Configuration requirements` from [this](https://docs.azure.cn/en-us/azure-sql/managed-instance/failover-group-configure-sql-mi?tabs=azure-portal%2Cazure-portal-modify%2Cazure-powershell-manage#configuration-requirements) document for more information.

`Managed instances should be deployed to [paired regions](https://docs.azure.cn/en-us/availability-zones/cross-region-replication-azure) for performance reasons. Managed instances that reside in geo-paired regions benefit from a significantly higher geo-replication speed compared to unpaired regions.`

Test result:
PASS: TestAccMsSqlManagedInstanceFailoverGroup_update (8437.15s)